src/components/buttons/Button.jsx

```diff
@@ -0,0 +1,159 @@
+// @flow
+
+import * as React from 'react';
+import type {Node} from 'react';
+import classNames from 'classnames';
+import {BUTTON_SIZE, BUTTON_TYPE} from './ButtonConsts';
+
+type ButtonTypeType =
+  | 'primary'
+  | 'primary-inverted'
+  | 'primary-blue'
+  | 'primary-mint'
+  | 'secondary'
+  | 'link-button'
+  | 'link-button-inverted'
+  | 'link-button-peach'
+  | 'link-button-mustard'
+  | 'destructive'
+  | 'warning'
+  | 'facebook';
+
+type ButtonSizeType =
+  | 'large'
+  | 'medium'
+  | 'small';
+
+export {BUTTON_SIZE, BUTTON_TYPE};
```

Comment:
Do we need that since we relay on flow?

Reply:
hmm, I think, I would need to have a look at html sketchapp as well, as I export all the buttons using it. not sure if this change would not require changes in export logic as well. Maybe we can have a look on it together?